### PR TITLE
Define new error type for heartbeat

### PR DESCRIPTION
**Pull request by DaikiMaekawa**

### CODE REVIEW

**Comment by DaikiMaekawa** [comment]
@at-wat PTAL

File: src/odometry.c [hunk 1]
```diff
@@ -281,19 +282,42 @@ void process_int(
...
+      }
+      if (value & ERROR_HEARTBEAT)
+      {
+        yprintf(OUTPUT_LV_ERROR, "ERROR_HEARTBEAT ");
+        unknown_err = 0;
+      }
 
       if (value != ERROR_NONE)
+      {
+        if (unknown_err)
```
Comment (at-wat) [inline src/odometry.c]: Unknown error bits can also be set even if known error bits are set.
Clearing known error bits in above if-blocks and checking remaining bits should work.

File: src/odometry.c [hunk 2]
```diff
@@ -281,19 +282,42 @@ void process_int(
...
+          yprintf(OUTPUT_LV_ERROR, "ERROR_UNKNOWN ");
```
Comment (at-wat) [inline src/odometry.c]:
```suggestion
          yprintf(OUTPUT_LV_ERROR, "ERROR_UNKNOWN(%x) ", value);
```

File: src/odometry.c (hunk 1 above)
Comment (DaikiMaekawa) [reply src/odometry.c]: done.

**Comment by at-wat** [comment]
The code looks good to me.

I'd like to make a new release before adding the heartbeat feature and it requires that [a fix in ros/catkin](https://github.com/ros/catkin/pull/1000) is applied to the buildfarm environment to test it.
Please let me hold this PR until they are ready.

File: include/shvel-param.h [hunk 3]
```diff
@@ -102,7 +102,8 @@ typedef enum
   ERROR_LOW_VOLTAGE = 0x0001,
   ERROR_HALL_SEQ = 0x0002,
   ERROR_HALL_ENC = 0x0004,
-  ERROR_WATCHDOG = 0x0008
+  ERROR_WATCHDOG = 0x0008,
+  ERROR_HEARTBEAT = 0x0010
```
Comment (at-wat) [inline include/shvel-param.h]: https://github.com/T-frog/tf-2md3-firmware/blob/ac0d851e6811b4f8c678ed9d6458f1ac1d80ac7c/tfrog-motordriver/controlVelocity.h#L37-L38
please make it `0x0040`

**Comment by at-wat** [comment]
Closing due to inactivity. Feel free to reopen if needed.

**Pull request closed without merging**